va/va.c

```diff
@@ -380,19 +392,14 @@ static VAStatus va_getDriverNameByIndex(VADisplay dpy, char **driver_name, int c
         va_infoMessage(dpy, "User requested driver '%s'\n", *driver_name);
         return VA_STATUS_SUCCESS;
     } else if (driver_name_env && (geteuid() == getuid())) {
+        if(!(*driver_name))
```

Comment:
I'm almost sure this is wrong, unless something very clever is involved here: you seem to free `*driver_name` when it is 0...

Reply:
sorry about it, I made a mistake, thanks you